### DISCUSSION TITLE
Accessing the model constants of Spallart Allamaras model through Ansys Fluent Core

### DISCUSSION THREAD

**Discussion by angadsahnii** [ansys/pyfluent#2542] [Q&A]
Hi, 

I want to edit the model constants of the Spalart Allamars model in Python. Is there a specific command for this since I am able to access all the other options except for the model constants through Python?

**Comment by mkundu1** [comment]
The turbulence model constants are not available in PyFluent at present. They will be added in future releases of Fluent.

The only possible workaround currently is to save a case or settings file with the modified settings from Fluent GUI and use that case or settings file in the PyFluent session.

**Reply by angadsahnii** [reply]
Thank you.
